dask/dataframe/io/tests/test_json.py

```diff
@@ -34,6 +34,10 @@ def test_read_json_basic(orient):
             out.columns = list(df.columns)
         assert_eq(out, df)
 
+        with pytest.raises(ValueError) as e:
```

Comment:
Let's split this into its own test specifically for `meta`.

I think the rough goal of this test should be to

1. create a JSON string that `dd.read_json` interprets incorrectly by default. The easiest way to do this is probably to have a column where the first partition is all ints, and the second partition is a mix of ints and NaNs.
2. Let the user specify `meta=...`, with the correct metadata.
3. Ensure that that *doesn't'* raise.

LMK if you need help with writing this, though it could be a little while before I can take a closer look.

Also, some test failures in https://travis-ci.org/dask/dask/jobs/503501340. I haven't looked too closely yet.

Reply:
By split into its own test, do you mean a new module for meta or a new file `test_meta`.

Can you show me an example of how user specified meta would look like? I was having confusion with that.
Also, I don't completely understand the CI error.